src/main/resources/db/changelog/v.0.5/db.changelog-table-location.yaml

```diff
@@ -0,0 +1,46 @@
+databaseChangeLog:
+  - changeSet:
+      id: 24042021-create-table-location
+      author: olga-evdokimova
+      changes:
+        - createTable:
+            tableName: location
+            columns:
+              - column:
+                  name: location_id
+                  type: UUID
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: l_county_id
+                  type: UUID
+                  constraints:
+                    nullable: false
+              - column:
+                  name: l_city_id
+                  type: UUID
+                  constraints:
+                    nullable: false
```

Comment:
probably it can be null since internship can be applied on whole country, must not particular city.

Reply:
Done